README.md

```diff
@@ -1,8 +1,6 @@
-# opa4j
+# JARL
```

Comment:
Should it be `JARL` or `Jarl`? 🤔

Reply:
I'm in favor of Jarl... it's more of a cool name than an acronym really :)

Reply:
Yea, the acronym is pretty forced 😄